relay-general/src/pii/attachments.rs

```diff
@@ -365,6 +365,13 @@ pub struct PiiAttachmentsProcessor<'a> {
     root_state: ProcessingState<'static>,
 }
 
+/// Which encodings to scrub for `scrub_bytes`.
+pub enum ScrubEncodings {
```

Comment:
I find this and especially it's use horrible :(

Reply:
Not sure that's worth it for now.  What I really struggled with was to make `scrub_bytes()` nice to implement for all these combinations.  I just did it the dumb, duplicating way for now but it certainly doesn't make me happy.

Reply:
I think it's worth it... I don't see much work in deduplicating it?